src/input_parsers/onnx_example/main.cpp

```diff
@@ -0,0 +1,30 @@
+/*********************                                                        */
+/*! \file main.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Matthew Daggitt
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ **/
+
+#include "Engine.h"
+#include "InputQuery.h"
+#include "MpsParser.h"
+#include <cstdio>
+
+int main( int argc, char *argv[] )
+{
+    return 0;
```

Comment:
This could be nearly identical to the AcasParser, right?

Reply:
Well no because we can't evaluate the error as we can't run inference on the Onnx network (would need the `onnxruntime` library to do that).

Leaving that aside, I don't really understand the purpose of these example files? Why do they exist in addition to the test suite?